internal/cmd/kafka/utils.go

```diff
@@ -107,9 +106,6 @@ func getKafkaRestProxyAndLkcId(c *pcmd.AuthenticatedStateFlagCommand) (*pcmd.Kaf
 }
 
 func isClusterResizeInProgress(currentCluster *cmkv2.CmkV2Cluster) error {
-	if currentCluster.Status.Phase == ccloudv2.StatusProvisioning {
-		return errors.New(errors.KafkaClusterStillProvisioningErrorMsg)
-	}
```

Comment:
Is it safe to remove this check; are we sure no other function is relying on this check?

Reply:
Yep, this is safe to remove!